core/src/main/java/ch/cyberduck/core/preferences/Preferences.java

```diff
@@ -555,6 +555,7 @@ protected void setFactories() {
         this.setDefault("factory.connectiontimeout.class", DefaultConnectionTimeout.class.getName());
         this.setDefault("factory.authorizationcodeprovider.class", "ch.cyberduck.core.oauth.BrowserOAuth2AuthorizationCodeProvider");
         this.setDefault("factory.s3.pathcontainerservice.class", "ch.cyberduck.core.s3.S3PathContainerService");
+        this.setDefault("factory.oauthprefixservice.class", OAuthPrefixService.class.getName());
```

Comment:
@AliveDevil should this not be `ch.cyberduck.core.DefaultOAuthPrefixService` instead of the interface?

Reply:
Good catch. Should be.